requirements/requirements_tts.txt

```diff
@@ -5,7 +5,7 @@ jieba
 kornia
 librosa
 matplotlib
-nemo_text_processing==0.1.6rc0
+nemo_text_processing==0.1.7rc0
```

Comment:
similar question as above.

Reply:
This is a good question, maybe @ekmb knows?

Reply:
We should preferably not pin nemo text processing. It causes major issues when fixes need to be made to both at the same time. please consider removing this pin

Reply:
Agree.  what do guys working on TTS and nemo-text-processing think? 
@XuesongYang @redoctopus @ekmb @yzhang123

Reply:
I'm ok with removing the pin

Reply:
IIUC, so far the dependency of TN on the languages used by TTS is stable, so removing the pin will not make negative effects on our current TTS recipes. I am ok with removing the pin.